bin/run_docker.py

```diff
@@ -103,6 +103,62 @@ def handle_outputs(output_path: str, output_file_name: str, log_text: str):
     )
 
     return output_file, log_text
+def truncate_log_file(log_file_name: str,
+                      log_file_path: str,
+                      max_lines: int = 100) -> None:
+    """
+    Truncates the log file content to a maximum number of lines.
+
+    Args:
+        log_filename (str): The name of the log file to be truncated.
+        log_file_path (str): The path to the log file.
+        max_lines (int): The maximum number of lines to keep in the log file
+                         (default is 20).
+
+    """
+    # Construct the full path to the log file
+    log_file = os.path.join(log_file_path, log_file_name)
+
+    # Read the lines from the log file
+    with open(log_file, "r") as file:
+        lines = file.readlines()
+
+        # If the log file has more lines than the maximum, truncate it
+        if len(lines) > max_lines:
+            print("Truncating log file...")
+            with open(log_file, "w") as truncated_file:
+                # Write only the last max_lines lines to the log file
+                truncated_file.writelines(lines[-max_lines:])
+
+            # Print a message indicating that the log file has been truncated
+            print(f"Truncated log file {log_file} to a maximum of {max_lines} lines.")
+
+
+def get_last_lines(log_file_name, log_file_path, max_lines=100):
```

Comment:
I didn't see this being used somewhere - Is this something that'll be used later on?

Reply:
Yes, I think I'm going to put this PR back into draft until I implement the components discussed [here](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/pull/20#discussion_r1552263546).

Reply:
I'm no longer using this function.